### PR TITLE
Import old blog posts - part I.

### PR DESCRIPTION
**Pull request by lslezak**
- Import the old Agama blog posts from the YaST blog
- Just the first three posts to see how it works, more will follow
- Uses `image-zoom` plugin for showing full size images

### CODE REVIEW

File: blog/2024-02-08-year-of-agama.mdx [hunk 1]
```diff
@@ -0,0 +1,106 @@
+---
+date: 2024-02-08 12:00:00 +00:00
+title: The Year of Agama
+description: Take a look to the Agama roadmap for 2024
+tags:
+- project
+- systems-management
+- YaST
```
Comment (dgdavid) [inline blog/2024-02-08-year-of-agama.mdx:8]: Is `YaST` tag really needed. Nothing against YaST, btw.

File: blog/2024-02-08-year-of-agama.mdx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+storage configuration.
+
+The new interface aims to be easy enough for newcomers, as you can see in the following mock-up.
+But we know (open)SUSE users have big expectations in terms of customizing their setups. Thus,
+we updated the [document](https://github.com/agama-project/agama/blob/master/doc/storage_ui.md) that
+describes how the new interface will work and all the possibilities it will offer for those who
+decide to go beyond the initial proposal. If you want to check whether your basic needs would be
+covered, don't hesitate to take a look to the document and the extended mock-ups included on it.
+
+![Storage proposal](../static/img/blog/2024-02-08/storage.png)
```
Comment (dgdavid) [inline blog/2024-02-08-year-of-agama.mdx:70]: Just a though: I would like to take this migration as a turning point regard the _alt_ text we're adding to images. I mean, keep this as it is if it comes from a migration but would be nice to start using better text in future blog entries.

File: blog/2024-02-08-year-of-agama.mdx (hunk 1 above)
Comment (lslezak) [reply blog/2024-02-08-year-of-agama.mdx:8]: No idea, it was already in the original post: https://github.com/yast/yast.github.io/blob/65a5e6a16333c8c814002f82e9798d1658fa6077/_posts/2024-02-08-year-of-agama.md?plain=1#L11

File: blog/2024-02-08-year-of-agama.mdx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+---
+
+At the end of 2023 we announced Agama 7 stating that version is the first prototype we could consider
+to be (quoting ourselves) "functional enough", covering areas such as localization, network
+configuration, storage setup, authentication basis and some software selection. Now it's time to go
+deeper into every area... and we have a plan for that.
+
+{/* truncate */}
+
+## The Agama Roadmap for 2024 {#roadmap}
```
Comment (imobachgs) [inline blog/2024-02-08-year-of-agama.mdx:20]: I know you are just importing, but I wonder if we should adapt the capitalization. See https://github.com/agama-project/agama/pull/1633.

**Comment by dgdavid** [comment]
As stated in the PR description and discussed in IRC, this is just a migration of the old blog entries in the YaST blog. And it's fine. 

But maybe we could reconsidered a few things before merging it, 

* The tags: not sure if "YaST", "Agama", "Programming", "Project Systems", and "Managemen" tags provide any value in the new blog, honestly. What is more, the "Announcing" blog entries could have the "Release" tag instead.
* The titles: I'd change "Announcing Agama X" by "Release X" or so. "Agama" is too redundant here.

**Review by imobachgs — APPROVED** [review]

**Pull request merged**